src/test/RUNTESTS

```diff
@@ -354,7 +354,7 @@ testtype=check
 fstype=all
 testconfig="./testconfig.sh"
 killopt="-k 10s"
-runtest_timeout="3m"
+runtest_timeout="30m"
```

Comment:
Please do not change the global timeout for all tests. You can change the test type to long in the test case instead.

Reply:
roll-back done